### ISSUE TITLE
Error when Anki is opened with incremental-reading plugin: anki.errors.NotFoundError: Your database appears to be in an inconsistent state.

### ISSUE THREAD

**Issue by mbindewald** [tvhong/incremental-reading#21]
**Describe the bug**
Starting Anki with incremental reading add on causes errors, doesn't create new IR note type, and plugin functionality doesn't appear to be working at all.

**To Reproduce**
1. Enable Incremental Reading v4116 unofficial clone add on
2. Restart Anki

**Expected behavior**
Anki to start with IR add on working correctly, IR note type to be generated, and plugin functionality to work. 

**Logs / Screenshots**

```
Debug info:
Anki 2.1.62 (77dd1dc1) Python 3.9.15 Qt 6.4.3 PyQt 6.4.0
Platform: Windows-10-10.0.22621
Flags: frz=True ao=True sv=3
Add-ons, last update check: 2023-05-01 11:22:22

Caught exception:
Traceback (most recent call last):
  File "aqt.progress", line 118, in handler
  File "aqt.main", line 217, in on_window_init
  File "aqt.main", line 264, in setupProfileAfterWebviewsLoaded
  File "aqt.main", line 316, in setupProfile
  File "aqt.main", line 496, in loadProfile
  File "_aqt.hooks", line 3809, in __call__
  File "C:\Users\matth\AppData\Roaming\Anki2\addons21\999215520\main.py", line 62, in onProfileLoaded
    self.addModel()
  File "C:\Users\matth\AppData\Roaming\Anki2\addons21\999215520\main.py", line 170, in addModel
    mw.col.models.update_dict(model)
  File "anki.models", line 223, in update_dict
  File "anki._backend_generated", line 862, in update_notetype_legacy
  File "anki._backend", line 156, in _run_command
anki.errors.NotFoundError: Your database appears to be in an inconsistent state. Please use the Check Database action.
```

![image](https://user-images.githubusercontent.com/16680778/235593330-9f9b143b-779d-4854-b95f-48b34569946b.png)

**Specs (please complete the following information):**
- OS: Windows 11 Home 22621.1635
- Anki Version: Same issue on 2.1.61 and 2.1.62
- Incremental Reading Version: 4.11.6 (latest)

**Additional context**
- Issue occurs on a freshly created profile with ONLY this add on enabled.
- There are other add ons installed on the system that have generated card types etc in this profile, here's a list of installed add ons:
![image](https://user-images.githubusercontent.com/16680778/235594228-bc20ad09-deca-4e91-b695-90cb850eb8d0.png)
- Check database and check media actions indicate no errors

I haven't had time to dig into source code yet / haven't dug in to anything Anki related before but I'm happy to help investigate and provide any information, just let me know!

**Comment by mbindewald** [comment]
Adding a note type "IR3" manually prevents the error from happening when Anki is opened, but I get other errors when I attempt to import a url- I will try manually adding all of the correct fields / card types to the "IR3" note and see if that works.

**Comment by tvhong** [comment]
@mbindewald have you tried the suggestion - "Please use the Check Database action"?

**Comment by tvhong** [comment]
Hm.. this might be related to the most recent change I made to fix enabling Priority Queue: https://github.com/tvhong/incremental-reading/commit/34ed3bb0249a0ddcfebfc6e74c2adf182c265231

**Comment by tvhong** [comment]
It was really me breaking the software.

I have pushed a fix in v4.11.7 ([PR](https://github.com/tvhong/incremental-reading/commit/fd62a98fe121fe03c70e75b6c075ec7fb33bbd6c)). If you download the latest Add-on, this issue should be fixed.

**Comment by mbindewald** [comment]
It works! Thank you
